### ISSUE TITLE
support keras 2.0

### ISSUE THREAD

**Issue by cwmeijer** [NLeSC/mcfly#127]
See https://blog.keras.io/introducing-keras-2.html
"The API of most layers has significantly changed, in particular Dense, BatchNormalization, and all convolutional layers. However, we have set up compatibility interfaces so that your Keras 1 code will still run in Keras 2 without issues (while printing warnings to help you convert your layer calls to the new API)."
- [ ] Upgrade to keras 2.0
- [ ] Make sure everything still works
- [ ] Get rid of any warnings

**Comment by dafnevk** [comment]
I adjusted the modelgen.py module to use the new interfaces. The code will no longer work now with Keras v1.x
